kaldi fails to build on master

```
❯ nom-build -A kaldi
these derivations will be built:
  /nix/store/bm3r3nfv5w51dvqzfr5ix3piwvp8pq8f-kaldi-2020-12-26.drv
building '/nix/store/bm3r3nfv5w51dvqzfr5ix3piwvp8pq8f-kaldi-2020-12-26.drv' on 'ssh://hexa@phoibe.cysec.de'...
unpacking sources
unpacking source archive /nix/store/m7iyn2s0fb3s9wq2lvcbcyx6ywa1wihj-source
source root is source
patching sources
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/share/doc/kaldi -DCMAKE_INSTALL_INFODIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/include -DCMAKE_INSTALL_SBINDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/kmqs0wll31ylwbqkpmlgbjrn6ny3myik-binutils-2.35.1/bin/strip -DCMAKE_RANLIB=/nix/store/kmqs0wll31ylwbqkpmlgbjrn6ny3myik-binutils-2.35.1/bin/ranlib -DCMAKE_AR=/nix/store/kmqs0wll31ylwbqkpmlgbjrn6ny3myik-binutils-2.35.1/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/sznh4kw5j4g7wzspiwlr9llpq1gllm7w-kaldi-2020-12-26 -DKALDI_BUILD_TEST=off -DBUILD_SHARED_LIBS=on
-- The C compiler identification is GNU 10.3.0
-- The CXX compiler identification is GNU 10.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/2qmkkrkk2d1jvin6iy8yvfh7iaawyqwm-gcc-wrapper-10.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/2qmkkrkk2d1jvin6iy8yvfh7iaawyqwm-gcc-wrapper-10.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PythonInterp: /nix/store/dqxic3j7csd4ywn94n4smmnz55p039g3-python3-3.9.6/bin/python (found version "3.9.6")
-- Running gen_cmake_skeleton.py
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Looking for sgemm_
-- Looking for sgemm_ - found
-- Found BLAS: /nix/store/8ms61phzj3al5axdsciy8b0c5xxnxxgn-openblas-0.3.17/lib/libopenblas.so
-- Looking for cheev_
-- Looking for cheev_ - found
-- A library with LAPACK API found.
CUDA_TOOLKIT_ROOT_DIR not found or specified
-- Could NOT find CUDA (missing: CUDA_TOOLKIT_ROOT_DIR CUDA_NVCC_EXECUTABLE CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY)
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Configuring done
-- Generating done
-- Build files have been written to: /build/source/build/openfst-download
[ 11%] Creating directories for 'openfst'
[ 22%] Performing download step (git clone) for 'openfst'
[ 33%] Performing update step for 'openfst'
CMake Error at /build/source/build/openfst-download/openfst-prefix/tmp/openfst-gitupdate.cmake:25 (message):
  Failed to get the hash for HEAD:



make[2]: *** [CMakeFiles/openfst.dir/build.make:130: openfst-prefix/src/openfst-stamp/openfst-update] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/openfst.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
CMake Error at cmake/third_party/get_third_party.cmake:18 (message):
  Build step for openfst failed: 2
Call Stack (most recent call first):
  CMakeLists.txt:162 (get_third_party)


-- Configuring incomplete, errors occurred!
See also "/build/source/build/CMakeFiles/CMakeOutput.log".
See also "/build/source/build/CMakeFiles/CMakeError.log".
builder for '/nix/store/bm3r3nfv5w51dvqzfr5ix3piwvp8pq8f-kaldi-2020-12-26.drv' failed with exit code 1
```

cc @Mic92

--- a/pkgs/tools/audio/kaldi/default.nix
+++ b/pkgs/tools/audio/kaldi/default.nix
@@ -55,7 +55,7 @@ stdenv.mkDerivation {
       # https://github.com/kaldi-asr/kaldi/blob/c9d8b9ad3fef89237ba5517617d977b7d70a7ed5/cmake/VersionHelper.cmake#L8
       echo 0
     elif [[ "$1" == "rev-parse" ]]; then
-      echo 0bca6e76d24647427356dc242b0adbf3b5f1a8d9
+      echo ${openfst.rev}
       echo 0
     fi
     true

--- a/pkgs/tools/audio/kaldi/default.nix
+++ b/pkgs/tools/audio/kaldi/default.nix
@@ -54,6 +54,9 @@ stdenv.mkDerivation {
       # fix up this call:
       # https://github.com/kaldi-asr/kaldi/blob/c9d8b9ad3fef89237ba5517617d977b7d70a7ed5/cmake/VersionHelper.cmake#L8
       echo 0
+    elif [[ "$1" == "rev-parse" ]]; then
+      echo 0bca6e76d24647427356dc242b0adbf3b5f1a8d9
+      echo 0
     fi
     true
     EOF

